src/Sarif.WorkItems/SarifWorkItemContext.cs

```diff
@@ -12,13 +12,8 @@ public class SarifWorkItemContext : PropertiesDictionary
 
         public SarifWorkItemContext(SarifWorkItemContext initializer) : base(initializer) { }
 
-        internal void InitializeFromLog(SarifLog sarifLog)
-        {
-            // TODOD: Initialize source and non-source-controlled files here,
-            //        as determined by scraping all URLs associated with results.
-            // 
-            //        https://github.com/microsoft/sarif-sdk/issues/1752
-        }
+        internal void InitializeFromLog(SarifLog sarifLog) { }
```

Comment:
>InitializeFromLog [](start = 22, length = 17)

We can probably delete this initializer for now, since we haven't yet found a use for it. Fine for later. :)